pyroomacoustics/acoustics.py

```diff
@@ -29,9 +29,10 @@
 import math
 
 import numpy as np
+from scipy import signal
 from scipy.fftpack import dct
 from scipy.interpolate import interp1d
-from scipy.signal import butter, fftconvolve, sosfiltfilt
+from scipy.signal import butter, fftconvolve, hilbert, sosfiltfilt
```

Comment:
Haven't found the usage of Hilbert transform in this file.

Reply:
Thanks!